plugins/uefi/meson.build

```diff
@@ -7,7 +7,18 @@ if efi_os_dir != ''
   cargs += '-DEFI_OS_DIR="' + efi_os_dir + '"'
 endif
 
-install_data(['uefi.quirk'],
+run_target('uefi-update-quirk',
+  command: [
+    python3.path(),
+    join_paths(meson.current_source_dir(), 'generate-quirk.py'),
+    '/var/lib/fwupd/remotes.d/lvfs/metadata.xml.gz', 'uefi-generated.quirk'
+  ]
+)
+
+install_data([
+    'uefi.quirk',
+    'uefi-generated.quirk',
```

Comment:
Since this is coming from lvfs data, perhaps `uefi-lvfs-generated.quirk`?

Reply:
Sure, that works too, fixed.